### DISCUSSION TITLE
Mixing rules for DP/MM simulations

### DISCUSSION THREAD

**Discussion by sli259** [deepmodeling/deepmd-kit#2196] [Q&A]
Hi I am wondering where I can find the documents about the mixing rule for the DP/MM simulations. If I set up a simulation for a system with A and B, where A is defined by the DP model, and B uses traditional FF. What is the mixing rule between system A and B?

**Comment by wanghan-iapcm** [comment]
@njzjz will give nice suggestions.

**Comment by njzjz** [comment]
There is no automatic way to have the mixing rule. You need to train QM/MM interaction.
To train QM/MM interaction, you can refer [DPRc](https://docs.deepmodeling.com/projects/deepmd/en/v2.2.0.b0/model/dprc.html). The manual talks about training $\Delta E$, but the same method should also work for the energy $E$. However, in this way, you need to remove the MM energy from training data manually to ensure that the total energy does not contain L-J energies.
